src/fabric/src/fabric_view_all_docs.erl

```diff
@@ -59,7 +59,8 @@ go(DbName, Options, QueryArgs, Callback, Acc0) ->
         conflicts = Conflicts,
         skip = Skip,
         keys = Keys0,
-        extra = Extra
+        extra = Extra,
+        update_seq = UpdateSeq
```

Comment:
We should split the update of the record into two commits. 
1. add compatibility clause so with this commit we would be able to handle both kinds of records. This commit would convert new record to old one.
2. add new field and change conversion clause so it converts the old one into new.

The reason for the split is handling of mixed cluster case when some nodes are upgraded and some of them don't. 

This would allow us to do two stage upgrade of the cluster. First we would deploy version with first commit to all nodes. Then we wait a little bit and deploy the version with second commit.

This dance is usually needed for the cases when we upgrade records we send to other nodes. Which is the case for `#mrargs{}` .

Reply:
Thanks @iilyak. However, wants to double check with you whether we need two commits. Actually, the definition for `mrargs` was not changed. https://github.com/apache/couchdb/blob/master/src/couch_mrview/include/couch_mrview.hrl#L86.  The `update_seq` filed was already present in any nodes in mixed cluster. In my newly introduced codes,  the value of `update_seq` field is fetched here and used to check whether we need to return update_seq value or not. If request is sent to nodes which are not upgraded, update_seq information is not returned. Otherwise, for upgraded nodes, the update_seq information will be returned.